app/handlers.py

```diff
@@ -639,7 +639,13 @@ class AddressEditEN(Start):
             raise ex
 
         if case['region'][0] == 'N':
-            raise HTTPFound(self._request.app.router['StartLanguageOptionsNI:get'].url_for())
+            session["attributes"]["addressLine1"] = attributes["addressLine1"]
+            session["attributes"]["addressLine2"] = attributes["addressLine2"]
+            session["attributes"]["addressLine3"] = attributes["addressLine3"]
+            session["attributes"]["townName"] = attributes["townName"]
+            session["attributes"]["postcode"] = attributes["postcode"]
```

Comment:
Can this be `session["attributes"] = attributes`?

Reply:
No - attributes potentially contains other values, so I only wanted to update the ones that might be modified